lib/score.rb

```diff
@@ -0,0 +1,66 @@
+require_relative 'scorecard'
+require_relative 'round'
+
+class Score 
+
+    attr_reader :first_bowl, :second_bowl, :total_score, :score_card
+
+    def initialize
+        @score_card = []
+        @total_score = 0
+    end
+
+    def total(x, y)
+    @score_card << [x, y]
+    if @score_card.length > 1
+        if @score_card[-2][0] == 10
+            strike(x,y)
+        elsif @score_card[-2].sum == 10
+            @total_score += x
+            @total_score += x + y
+        else
+            p @score_card[-2]
+            @total_score += (x + y)
+        end
+    else 
+        @total_score += (x + y)
+    end
+    p "Your current total is #{@total_score}"
+    end
+
+    def strike(x, y)
+        #IF THREE IN A ROW, ADD THIRTY
+        if @score_card.length >= 3 && @score_card[-1][0] == 10 && @score_card[-2][0] == 10 && @score_card[-3][0] == 10
+            @total_score += 30
+        # IF TWO IN A ROW, ADD TWENTY
+        elsif
+            @score_card.length >= 2 && @score_card[-2][0] == 10 && @score_card[-1][0] == 10
```

Comment:
This could maybe be simplified a bit by drawing out these combinations of conditionals into methods that return booleans, and named something like 'is_three_in_a_row?', for example (not my best naming, but something like that). Descriptive methods that condense combinations of conditionals makes it easier for you to refactor logic like this, because you're thinking of descriptions of code / checks, rather than being faced with the logic itself, which isn't easy to understand just by reading.